plutus-core/plutus-ir/src/PlutusIR/Compiler/Types.hs

```diff
@@ -40,14 +41,23 @@ makeLenses ''PirTCConfig
 instance PLC.HasTypeCheckConfig (PirTCConfig uni fun) uni fun where
     typeCheckConfig = pirConfigTCConfig
 
-newtype CompilationOpts = CompilationOpts {
-    _coOptimize :: Bool
+data CompilationOpts = CompilationOpts {
+      _coOptimize                     :: Bool
+    , _coPedantic                     :: Bool
+    , _coVerbose                      :: Bool
+    , _coDebug                        :: Bool
+    , _coMaxSimplifierIterations      :: Int
+    -- Simplifier passes
+    , _coSimplifierUnwrapCancel       :: Bool
```

Comment:
I'd call these `coDoSimplifierUnwrapCancel` or something, so the polarity is obvious.